src/styles/_header.scss

```diff
@@ -1,3 +0,0 @@
-.logo {
```

Comment:
Were these `.logo` styles unused? Seems a little weird to me that this would get deleted, not sure I'm seeing them added back in elsewhere.

Reply:
Nvm, I see that these are no longer needed. Thanks for clarifying!

Reply:
😅